HTML4FUN

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta content="text/html; charset=utf-8">
+<title> Тестова сторінка  №1.</title>
+</head>
+<body>
+<table width="100%" border="1" cellspacing="1" cellpadding="1" bgcolor="#FFFF00">
```

Comment:
Move out all styles into separate css file. So that in this file will be only plain html.

Example:

HTML4FUN.html:
.....
<table class='container'>
....
</table>
......

styles.css:

.container {
 width: 100%;
 border: 1;
 cellspacing: 1;
 cellpadding: 1; 
 bgcolor: #FFFF00;
}